src/components/Facility/FacilityConfigure.tsx

```
@@ -1,166 +1,146 @@
-    });
+    updateFacility(formData);
   };
 
-  if (isLoading || loading) {
+  const form = useForm({
+    resolver: zodResolver(formSchema),
+    defaultValues: { middleware_address: "" },
+  });
+
+  useEffect(() => {
+    if (data && data.middleware_address) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)